server/meshmodel/aws-rds-controller/v1.4.3/v1.0.0/relationships/dbClusterSnapshot-to-dbCluster.json

```diff
@@ -0,0 +1,56 @@
+{
+    "schemaVersion": "relationships.meshery.io/v1alpha3",
+    "version": "v1.0.0",
+    "kind": "hierarchical",
+    "type": "sibling",
+    "subType": "backup",
```

Comment:
Here too it says backup

Reply:
While these aren't specifically supported, I don't think they hurt anything either (or they shouldn't, that is).